style.css

```diff
@@ -1,89 +1,746 @@
+    border: 1px solid #212121;
+  }
+
+  .works_title {
+    text-align: center;
+    margin-bottom: 80px;
+  }
+
+  .works_title span {
+    color: #1c1a19;
+    font-family: "Crete Round", serif;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 130%;
+    margin: 0;
+    height: 83px;
+  }
+
+  .works_content {
+    margin: 0;
+    padding: 0;
+  }
+
+  .works_content ul {
+    display: flex;
+    list-style-type: none;
+    justify-content: space-between;
+    margin-top: 0;
+    margin-bottom: 12px;
+    padding: 0;
+    gap: 12px;
+  }
+
+  .works {
+    display: grid;
+    grid-template-rows: auto auto auto auto;
+    background-color: white;
+    padding: 120px 24px 0 24px;
+  }
+
+  .works_content h2 {
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 138%;
+    margin-top: 0;
+    margin-bottom: 12px;
+  }
+
+  .works_content p {
+    margin-top: 0;
+    margin-bottom: 12px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    text-align: justify;
+    line-height: 150%;
+    color: #091e42;
+    display: grid;
+    order: 1;
+  }
+
+  .content_desc {
+    background-image: url("icons/professional.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .center_content {
+    display: grid;
+    grid-template-rows: auto auto auto auto;
+    background-color: white;
+    padding: 24px 16px 0 16px;
+  }
+
+  .content_desc h2 {
+    margin: 72px 16px 12px 16px;
+    font-family: "Crete Round", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 38px;
+    line-height: 138%;
+    color: #fff;
+  }
+
+  .content_desc p {
+    margin: 0 16px 12px 16px;
+    text-align: justify;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 150%;
+    color: #fff;
+  }
+
+  .content_desc ul {
+    display: flex;
+    list-style-type: none;
+    margin: 0 16px 22px 16px;
+    padding: 0;
+    gap: 12px;
+  }
+
+  .content_desc li {
+    margin: 2px;
+    background-color: rgba(255, 255, 255, 0.277);
+    padding: 10px;
+    color: #091e42;
+    font-size: 15px;
+    font-weight: 500;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+  }
+
+  .content_desc button {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    background: #ff6b00;
+    color: #fff;
+    border: 1px #ff6b00;
+    padding: 12px;
+    width: 100%;
+    height: 48px;
+    bottom: 0;
+  }
+
+  .contact_form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .contact_content p {
+    margin-top: 0;
+    font-family: "inter", sans-serif;
+    font-weight: 700;
+    font-style: normal;
+    font-size: 32px;
+    line-height: 138%;
+    text-align: center;
+    color: #172b4d;
+    margin-bottom: 119px;
+  }
```

Comment:
- [ ] Hey @Kanyorok, Kudos on implementing the project requirements but I would like you to make some adjustments to this section so that your work looks exactly like your chosen Figma template.  make `margin-top: 13px; ` `font-size: 28px;`

> Figma design
<img width="671" alt="Screenshot 2023-05-06 at 13 16 03" src="https://user-images.githubusercontent.com/71140133/236623972-402d0ec8-64d2-4037-84d0-7166a5201d52.png">

> Your design
<img width="1099" alt="Screenshot 2023-05-06 at 13 15 29" src="https://user-images.githubusercontent.com/71140133/236624002-47101433-2f39-4a10-adee-94e257fa01a6.png">